releases/2023.1/2023.1.md

```diff
@@ -0,0 +1,182 @@
+- Introduce `of_multi_table` NApp responsible for managing a multi-table OpenFlow pipeline
+- Evolve `flow_stats` into `kytos_stats` to support multi-table stats
+- Improve and stabilize `maintenance` NApp functionalities
+- Migrate `@rest` decorated routes from `werkzeug/flask` to `uvicorn/starlette`
+- Stabilize `sdntrace_cp` and general improvements
+- General UI fixes and enhancements
+- Specify requirements for `mef_eline` VLAN range feature and new Kytos-ng documentation
+- General core enhancements and endpoints OpenAPI validations
+- Upgrade and simplify core development dependencies
+
+In order to achieve the objectives of this version, the following enhancements were refined and delivered:
+
+### 1. Augment `mef_eline` to support `"untagged"` and `"any"` VLAN UNIs when provisioning EVCs
+
+`mef_eline` now supports provisioning EVCs with `"untagged"` and `"any"` VLAN UNIs. `"untagged"` will match any packet that doesn't have a VLAN header, whereas `"any"` will match any VLAN. `mef_eline` also supports asymmetric VLAN configurations on each UNI. If you need more information about the supported configuration combinations, there's a spreadsheet on [`mef_eline` PR 258](https://github.com/kytos-ng/mef_eline/pull/258) that documents the combinations for both intra and inter EVCs and which OpenFlow matches and actions will be performed.
+
+### 2. Introduce `of_multi_table` NApp responsible for managing a multi-table OpenFlow pipeline
+
+A new NApp [`of_multi_table`](https://github.com/kytos-ng/of_multi_table) has been developed to allow network operators to leverage OpenFlow multi-tables when that's beneficial for their network use case. `of_multi_table` is responsible for managing a multi-table OpenFlow pipeline. This NApp allows you to specify an OpenFlow pipeline composed of multiple tables, where you can also associate which table each NApp of the `kytos-ng` org should be pushing their flows. `of_multi_table` has been integrated with `mef_eline`, `of_lldp`, and `coloring`. Each NApp that pushes flows has a concept of `"table_group"`, which represents a set of flows with similar match criteria that is grouped in a particular table.
+
+### 3. Evolve `flow_stats` into `kytos_stats` to support multi-table stats
+
+In addition to supporting a multi-table OpenFlow pipeline on version `2023.1`, another new NApp [`kytos_stats`](https://github.com/kytos-ng/kytos_stats) has been developed, which is responsible for providing multi-table statistics. `kytos_stats` is a successor of `flow_stats`. It also supports the same functionalities that `flow_stats` supports, except that `flow_stats` route prefixes were renamed. On version `2023.1` you can still use both if you're using `flow_stats` and still need the same API routes. In the future, `flow_stats` will no longer be maintained by Kytos-ng team, so you should plan to migrate to `kytos_stats`.
+
+
+### 4. Improve and stabilize `maintenance` NApp functionalities
```

Comment:
@Ktmi I closed https://github.com/kytos-ng/kytos-ng.github.io/issues/56, feel free to augment anything else regarding `maintenance` if you wish.

Reply:
This looks good here.

Reply:
Right. Let's close this thread then.